devtools-ci-index.yaml

```diff
@@ -4244,3 +4244,20 @@
             ci_project: 'devtools'
             ci_cmd: '/bin/bash .cico/build.sh'
             timeout: '20m'
+        - '{ci_project}-{git_repo}-build-master':
+            git_organization: fabric8-analytics
+            git_repo: fabric8-analytics-release-monitor
+            ci_project: 'devtools'
+            ci_cmd: '/bin/bash cico_build_deploy.sh'
+            skip_deploy: 1
```

Comment:
The template `{ci_project}-{git_repo}-build-master` does not define `skip_deploy`. Perhaps you meant `{ci_project}-{git_repo}-f8a-build-master`?